AdobeStockImageAdminUi/view/adminhtml/web/js/components/grid/column/image-preview.js

```diff
@@ -332,6 +348,52 @@ define([
             });
         },
 
+        /**
+         * Locate downloaded image in media browser
+         *
+         * @param record
+         */
+        locate: function (record) {
+            $.ajaxSetup({async: false});
```

Comment:
Can you please extract this method to a separate JS file (`image-preview.js` is becoming a really long file and we are trying to limit it's responsibility to image preview rendering only).

I'd create `AdobeStockImageAdminUi/view/adminhtml/web/js/media-gallery.js` returning simple object:

```
define([
...
], function (...) {
        'use strict';

        return {
            jsTreeRootFolderName: 'Storage Root',
            jsTreeFolderNameMaxLength: 20,
           selectImage: function (path) {
              ...
           }
        }
    }
```